.helmignore

```diff
@@ -1,50 +1,47 @@
-# Copyright 2018-2021 Crown Copyright
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-# Patterns to ignore when building packages.
-# This supports shell glob matching, relative path matching, and
-# negation (prefixed with !). Only one pattern per line.
-.DS_Store
-# Common VCS dirs
-.git/
-.gitignore
-*.tmp
-*~
-Jenkinsfile
-mvn_dependency_tree.txt
-.hadolint.yaml
-pom.xml
-*.md
-velocity*
-# Various IDEs
-.project
-.idea/
-*.iml
-*.tmproj
-.vscode/
-# Non-service directories
-/logos/
-/target/
-/licenses/
-/code-style/
-*.lock
-# Previous helm packages
-Palisade-examples-*.tgz
-# Add the directory for every service module here
-/deployment/
-/example-library/
-/example-runner/
-/hr-data-generator/
-/performance/
-/resources/

```

Comment:
you've deleted these and not readded. check if theyre still needed

Reply:
I think they are needed again. I have added them back in.